base/multidimensional.jl

```diff
@@ -576,7 +576,7 @@ end
 
 @generated function findn(A::AbstractArray{T,N}) where {T,N}
     quote
-        nnzA = countnz(A)
+        nnzA = count(!iszero, A)
```

Comment:
This is inconsistent with the `findn` definition above and with its docstring, which states "determined by `A[i]!=0`".